tensorflow/compiler/mlir/tosa/transforms/legalize_utils.cc

```diff
@@ -691,5 +691,41 @@ LogicalResult ApplyPatternsWithShapeResolution(
   return success();
 }
 
+void TrimQuantizedIntegerRangeMin(UniformQuantizedType dtype,
+                               int64_t& val_min) {
+  // Ensure that zero-point adjusted clamp values are valid for the data size.
+  uint32_t bitwidth = dtype.getStorageTypeIntegralWidth();
+
+  // Signed version
+  if (dtype.isSigned()) {
```

Comment:
Change the structure to:

```
if (dtype.isUnsigned()) {
  <do things>
 return;
}

<do other things>
```

It feels like a minor difference however integers are signed, unsigned, or signless. Signless integers we default to signed behavior.  We also avoid including the else if there can be an early return which reduces the number of blocks.
```